tock/tock/static/js/components/filter_report_table.js

```diff
@@ -0,0 +1,46 @@
+(function () {
```

Comment:
I'd like to make this whole file into a more general filtering function, but I figured this was a first pass, at least.